plugins/generator-1.19.4/forge-1.19.4/templates/elementinits/entities.java.ftl

```diff
@@ -57,7 +57,7 @@ package ${package}.init;
 						);
 			<#if entity.hasCustomProjectile()>
 			public static final RegistryObject<EntityType<${entity.getModElement().getName()}EntityProjectile>> ${entity.getModElement().getRegistryNameUpper()}_PROJECTILE =
-				register("projectile_${entity.getModElement().getRegistryName()}", EntityType.Builder.<${entity.getModElement().getName()}EntityProjectile>
+				register("${entity.getModElement().getRegistryName()}", EntityType.Builder.<${entity.getModElement().getName()}EntityProjectile>

```

Comment:
I doubt this PR was tested. Custom entities with custom projectiles will now register two entities with the same registry name.

Please fix, and retest the whole PR

Reply:
Ok

Reply:
Add the projectile_ prefix back to the custom entity projectile and it should be good. That projectile is only registered if your ranged entity does not use a projectile element as its projectile.

Reply:
Yes